README.md

```diff
@@ -15,6 +15,13 @@ Epiphany at its core is a full automation of Kubernetes and Docker plus addition
 - Vault (MVP) for protecting secrets and other sensitive data
 - Helm as package manager for Kubernetes
 
+---
+**NOTE**
```

Comment:
I would also not put it straight here. Lets only keep the in know issues message that Istio and vault or not working properly with latest K8s and most likely not fixing it because it will probably be deprecated in feature release.